lib/rails_response_dumper/option_parser.rb

```diff
@@ -7,6 +7,16 @@ def self.parse_options!
     options = {}
 
     OptionParser.new do |opts|
+      opts.banner += ' [files or directories]'
```

Comment:
Does specifying a directory work?

Might to useful to emphasize that this is a glob.

```suggestion
      opts.banner += ' [glob]'
```